config/samples/logging_v1_output.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: logging.openshift.io/v1
```

Comment:
Something is of here since we do not have a kind of "Output"  these samples should be removed

Reply:
ok